src/devices/Bmx280/BmxBase.cs

```diff
@@ -391,7 +382,7 @@ internal ushort Read16BitsFromRegister(byte register)
         /// <returns>
         ///  Value from register
         /// </returns>
-        private uint Read24BitsFromRegister(byte register)
+        internal uint Read24BitsFromRegister(byte register)
```

Comment:
we should add some helpers for 24-bit numbers - I've seen this like 3 times already at least and every one does it differently